Library/Formula/open-scene-graph.rb

```diff
@@ -53,6 +61,10 @@ def install
     args = std_cmake_args
     args << "-DBUILD_DOCUMENTATION=" + ((build.with? "docs") ? "ON" : "OFF")
 
+    # Fixes Xcode 7.1 failure due to warning in system Framework
+    # https://github.com/Homebrew/homebrew/issues/46372
+    args << "-DCMAKE_CXX_FLAGS='-Wno-c++11-narrowing'"
```

Comment:
Those embedded single quotes are not necessary. Luckily, `cmake` ignores them, so they do not harm. But I'd still vote for getting rid of them.